styles.css

```diff
@@ -0,0 +1,174 @@
+
+.tool-bar {
+  position: absolute;
+  width: 375px;
+  height: 92px;
+  left: 0;
+  top: 0;
+  background: #3c3a39;
+}
+
+.head-line {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+  gap: 12px;
+  position: absolute;
+  width: 375px;
+  top: 92px;
+  background: #1c1a19;
+  height: 590px;
+  left: 0;
+}
+
+.bars-status-dark {
```

Comment:
- [ ] Talking about your styles and the design, I would like to inform you that you forgot to reference your stylesheet in the HTML document. Therefore, I didn't have a chance to review what you have done so far. Look at your HTML in the `head` 👇 
![image](https://user-images.githubusercontent.com/90222110/176961167-89d6fb1f-c044-453c-ad05-ca2d4d81b0f3.png)
Please reference your stylesheet so that your styles are applied.